test/app.test.ts

```diff
@@ -0,0 +1,118 @@
+import * as chai from "chai";
+import { BigNumber, ethers } from "ethers";
+import { JsonConvert } from "json2typescript";
+import { AppModule, CollectionModuleMetadata, ThirdwebSDK } from "../src/index";
+
+global.fetch = require("node-fetch");
+
+const RPC_URL = "https://matic-mumbai.chainstacklabs.com";
+
+describe("App Module", async () => {
+  let sdk: ThirdwebSDK;
+  let appModule: AppModule;
+
+  beforeEach(async () => {
+    sdk = new ThirdwebSDK(
+      new ethers.Wallet(process.env.PKEY, ethers.getDefaultProvider(RPC_URL)),
+    );
+    appModule = sdk.getAppModule("0xA47220197e8c7F7ec462989Ca992b706747B77A8");
+  });
+
+  it("should serialize metadata correctly", async () => {
+    const tests: {
+      expected: any;
+      test: CollectionModuleMetadata;
+      type: any;
+    }[] = [
+      {
+        expected: {
+          name: "Testing",
+          description: "Test description",
+          external_link: "https://google.com",
+          seller_fee_basis_points: 100,
+        },
+        test: {
+          name: "Testing",
+          description: "Test description",
+          externalLink: "https://google.com",
+          sellerFeeBasisPoints: 100,
+        },
+        type: CollectionModuleMetadata,
+      },
+      {
+        expected: {
+          name: "Testing",
+          description: "Test description",
+          external_link: "https://google.com",
+          seller_fee_basis_points: 100,
+          fee_recipient: "0x0",
+          image: "test",
+        },
+        test: {
+          name: "Testing",
+          description: "Test description",
+          externalLink: "https://google.com",
+          sellerFeeBasisPoints: 100,
+          feeRecipient: "0x0",
+          image: "test",
+        } as CollectionModuleMetadata,
+        type: CollectionModuleMetadata,
+      },
+      {
+        expected: {
+          name: "Testing",
+        },
+        test: {
+          name: "Testing",
+        } as CollectionModuleMetadata,
+        type: CollectionModuleMetadata,
+      },
+    ];
+
+    const jsonConvert = new JsonConvert();
+    for (const test of tests) {
+      const result = jsonConvert.serializeObject(test.test, test.type);
+      chai.assert.deepEqual(result, test.expected);
+    }
+  });
+
+  it.skip("should deploy a collection module successfully", async () => {
+    const module = await appModule.deployCollectionModule({
+      name: "Testing module from SDK",
+      sellerFeeBasisPoints: 1000,
+      image:
+        "https://pbs.twimg.com/profile_images/1433508973215367176/XBCfBn3g_400x400.jpg",
+    });
+
+    console.log("DEPLOYED MODULE =", module);
+  });
+
+  it.skip("should deploy a splits module successfully", async () => {
```

Comment:
@jakeloo @jnsdls 

So its important that we use `shares` here and not %'ages

I tried to communicate this as best as I could in the `NewSplitRecipient.ts` file, but it might be a bit confusing.

Let's just make sure we highlight very clearly on the guide and people expect that `recipient percentage = recipient_shares / sum(all shares)`